packages/lab/src/Flow/types/flow.ts

```diff
@@ -8,57 +8,40 @@ import { HvColorAny } from "@hitachivantara/uikit-styles";
 
 // Node types
 
-type NodeExtras<GroupId extends keyof any = string, NodeData = any> = {
-  meta?: HvFlowNodeTypeMeta<GroupId, NodeData>;
-};
-
 /** HvFlowNode component type. @extends React.FC */
-export interface HvFlowNodeFC<
-  GroupId extends keyof any = string,
-  NodeData = any
-> extends FC<NodeProps<NodeData>>,
-    NodeExtras<GroupId, NodeData> {}
-
-export interface HvFlowNodeComponentClass<
-  GroupId extends keyof any = string,
-  NodeData = any
-> extends ComponentClass<NodeProps>,
-    NodeExtras<GroupId, NodeData> {}
-
-export type HvFlowNodeComponentType<
-  GroupId extends keyof any = string,
-  NodeData = any
-> =
-  | HvFlowNodeComponentClass<GroupId, NodeData>
-  | HvFlowNodeFC<GroupId, NodeData>;
-
-export type HvFlowNodeTypes<
-  GroupId extends keyof any = string,
-  NodeData = any
-> = Record<string, HvFlowNodeComponentType<GroupId, NodeData>>;
+export interface HvFlowNodeFC<NodeData = any> extends FC<NodeProps<NodeData>> {}
+
+export interface HvFlowNodeComponentClass extends ComponentClass<NodeProps> {}
+
+export type HvFlowNodeComponentType<NodeData = any> =
+  | HvFlowNodeComponentClass
+  | HvFlowNodeFC<NodeData>;
+
+export type HvFlowNodeTypes<NodeData = any> = Record<
+  string,
+  HvFlowNodeComponentType<NodeData>
+>;
+
+export interface HvFlowGroupItem {
```

Comment:
Shouldn't we be able to configure other Node properties? eg. subtitle, params, etc?

Reply:
We already get the subtitle from the item (line 95, `Node.tsx`):

```
const nodeSubtitle = (groupItem && nodeGroup?.items?.[groupItem]?.label) || subtitle;
```

As for adding `params`, is there an advantage to adding it to the item structure that I'm not seeing? If not, I think we should leave the node's configuration at the node level. We have `data` because we need to add it to the node when we are dropping the node on the canvas.

Reply:
What if nodes different "node configurations" need different `params`?
DA actually was doing this at the config level (though avoidable)

Reply:
Label/Subtitle is addressed through `data.customNodeLabel`